src/Surfnet/ServiceProviderDashboard/Infrastructure/Jira/Factory/IssueFieldFactory.php

```diff
@@ -107,6 +123,22 @@ private function translateDescription(Ticket $ticket): string
         ]);
     }
 
+    private function translateConnectionRequestDescriptions(Ticket $ticket): string
+    {
+        $translation = '';
+        foreach ($ticket->getConnectionRequests() ?? [] as $connectionRequest) {
+            $translation .= $this->translator->trans($ticket->getDescriptionTranslationKey(), [
+                '%applicant_name%' => $ticket->getApplicantName(),
+                '%applicant_email%' =>  $ticket->getApplicantEmail(),
+                '%entity_name%' => $ticket->getEntityName(),
+                '%institution_name%' => $connectionRequest->institution,
+                '%contact_name%' => $connectionRequest->name,
+                '%contact_email%' => $connectionRequest->email
+            ]);
+        }
```

Comment:
I don't think this will work. The applicant data and the entity name should not be in the foreach loop right? They are displayed once.

Wouldn't it make sense to add a single `%connection_requests%` parameter. And create that in a HTML friendly fashion based on the available connection requests? I recon you could even use a twig template to achieve this goal.

Reply:
You are right. The applicant (display once) and institution data (display multiple) have to be split up.